docs/api.md

```diff
@@ -1507,6 +1508,19 @@ If the name is empty, returns the id attribute instead.
 #### frame.parentFrame()
 - returns: <[Frame]> Returns parent frame, if any. Detached frames and main frames return `null`.
 
+#### frame.select(selector, ...values)
+- `selector` <[string]> A [selector] to query page for
```

Comment:
nit: s/page/frame

Reply:
@aslushnikov I couldnt get you here.

Reply:
@suvratjain1995 sorry, I meant that "frame" should be used instead of "page".